ros_buildfarm/common.py

```diff
@@ -298,10 +300,13 @@ def get_short_os_code_name(os_code_name):
 
 def get_short_arch(arch):
     arch_mappings = {
+        'aarch64': 'v8',
         'amd64': '64',
         'arm64': 'v8',
         'armhf': 'hf',
+        'armhfp': 'hf',
         'i386': '32',
+        'x86_64': '64',
```

Comment:
I assume we need these custom keys (even though they all mean the same as existing ones) and can't map them to existing keys instead?

Reply:
If we use the existing ones, I'll need to set up another layer of mapping to convert between them because the ones used in Debian are meaningless to RPM repositories. I can take that route if you prefer.